src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+import { wait } from '../../utils/fetchClient';
+import { Error } from '../../types/Error';
+
+type Props = {
+  errors: Error[];
+  setErrors: (cb: (prev: Error[]) => Error[]) => void;
+};
+
+export const Notification: React.FC<Props> = ({ errors, setErrors }) => {
+  const [isHidden, setIsHidden] = useState(false);
+  const { title, isDanger } = errors[0];
```

Comment:
Accessing 'errors[0]' directly may cause an error if the 'errors' array is empty. It's better to destructure the first element with a default value or check if the array is not empty before accessing it.